src/VisualStudio/IntegrationTests/VisualBasic/BasicBuild.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Roslyn.VisualStudio.Test.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.Integration.UnitTests
+{
+    [Collection(nameof(SharedIntegrationHost))]
+    public class BasicBuild
+    {
+        private IntegrationHost _host;
+
+        public BasicBuild(IntegrationHost host)
+        {
+            _host = host;
+            _host.Initialize();
```

Comment:
Because:
* Test A runs
* Test A fails
* Test B runs
* Test B needs to reset the shared host because Test A failed

This allows us to reuse a VS Host Process, when everything is going well, but tear it down and start fresh if something goes awry.

And as far as I know, xUnit doesn't provide a mechanism to say 'Give me a new instance of the shared fixture'.

I imagine I could probably modify the fixture to expose the `IntegrationHost` via a property, and have the property initialize the host if required (I do already know the fixture needs to be modified to support a set of hosts, if we want parallelization is properly supported, if that makes any difference on this)....